common/entity/Person.ts

```diff
@@ -58,6 +58,13 @@ export abstract class Person {
 
     @Column({ nullable: true, type: 'json' })
     notificationPreferences: any;
+
+    @Column({
+        type: 'timestamp',
+        nullable: true,
+        default: new Date(),
+    })
+    lastLogin?: Date;
```

Comment:
I think we also need a migration to add the field in production

Reply:
I always forget those 😅

Reply:
Why is this nullable while the migration creates a NOT NULL column?

Reply:
So you think there shouldn't be a default value at all? I think having a default makes sense, as the redaction job can just work with the default values, such that accounts that don't log in after the migration will be redacted in 3 months. 🙂 
I agree though, it's an oversight to have unmatching properties.

Reply:
I think all the current users should have `now()` as default as we don't have any other source to firgure out their last login.
For new users we should think about it tho. If allowing null, we could see how many users never logged in. Maybe this is also possible with other metrics :-D